src/app/containers/FrontPageSection/index.jsx

```diff
@@ -103,19 +126,21 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
         {group.strapline.name}
       </SectionLabel>
       {items.length > 1 ? (
-        <MarginWrapper addMargin={!isFirstSection} as={StoryPromoUl}>
-          {items.map((item, index) => (
-            <StoryPromoLi key={item.id}>
-              <StoryPromoComponent
-                item={item}
-                sectionNumber={sectionNumber}
-                storyNumber={index}
-              />
-            </StoryPromoLi>
-          ))}
+        <MarginWrapper firstSection={isFirstSection}>
+          <StoryPromoUl>
+            {items.map((item, index) => (
+              <StoryPromoLi key={item.id}>
+                <StoryPromoComponent
+                  item={item}
+                  sectionNumber={sectionNumber}
+                  storyNumber={index}
+                />
+              </StoryPromoLi>
+            ))}
+          </StoryPromoUl>
         </MarginWrapper>
       ) : (
-        <MarginWrapper addMargin={!isFirstSection}>
+        <MarginWrapper firstSection={isFirstSection} oneItem>
```

Comment:
I'm assuming `oneItem` is a boolean, if so could you rename it to `isOneItem`?

Reply:
That props sets always to true (oneItem={true}), we are not getting the value above, that's why I name it like this because we pass it just when there is one promo.